v1/metrics.yaml

```diff
@@ -151,10 +163,22 @@ paths:
           description: Error
           schema:
             $ref: '#/definitions/problem'
+      x-route-filters:
+        - type: default
+          name: ratelimit_route
+          options:
+            limits:
+              internal: 100
+              external: 100
+            log_only: true
       x-request-handler:
         - get_from_backend:
             request:
               uri: '{{options.host}}/pageviews/aggregate/{project}/{access}/{agent}/{granularity}/{start}/{end}'
+              headers:
+                cache-control: '{{cache-control}}'
+                if-unmodified-since: '{{if-unmodified-since}}'
```

Comment:
In general we use `cache-control` to force a rerender, and `if-unmodified-since` to skip a forced rerender if requests come out of order. Since AQS data is practically immutable, you can't rerender, so unlikely you need them.